src/paradigma/tremor/tremor_analysis.py

```diff
@@ -47,3 +49,57 @@ def extract_tremor_features_io(input_path: Union[str, Path], output_path: Union[
     write_df_data(metadata_time, metadata_samples, output_path, 'tremor_meta.json', df_windowed)
 
 
+def detect_tremor(df: pd.DataFrame, config: TremorDetectionConfig, path_to_classifier_input: Union[str, Path]) -> pd.DataFrame:
+   
+    # Initialize the classifier
+    coefficients = np.loadtxt(os.path.join(path_to_classifier_input, config.coefficients_file_name))
+    threshold = np.loadtxt(os.path.join(path_to_classifier_input, config.thresholds_file_name))
+
+    # Scale the mfcc's
+    mean_scaling = np.loadtxt(os.path.join(path_to_classifier_input, config.mean_scaling_file_name))
+    std_scaling = np.loadtxt(os.path.join(path_to_classifier_input, config.std_scaling_file_name))
+    mfcc = df.loc[:, df.columns.str.startswith('mfcc')]
+    mfcc_scaled = (mfcc-mean_scaling)/std_scaling
```

Comment:
Normalizeer je de MFCCs niet ook al bij feature extraction? Ik snap dat je hier normaliseert dmv een scaler gefit op een hele dataset, maar dan  ben ik benieuwd wat de toegevoegde waarde van het schalen binnen het maken van de MFCCs is.

Reply:
Ik snap niet precies wat je bedoelt met schalen binnen het maken van de MFCCs, kunnen we het maandag wel over hebben als we naar de MFCCs kijken.